src/modules/internet/index.ts

```diff
@@ -804,4 +805,104 @@ export class InternetModule extends ModuleBase {
       this.faker.definitions.internet.emoji[emojiType]
     );
   }
+
+  /**
+   * Generates a random JWT (JSON Web Token) Algorithm.
+   *
+   * @see faker.internet.jwt(): For generating random JWT (JSON Web Token).
+   *
+   * @example
+   * faker.internet.jwtAlgorithm() // 'HS256'
+   * faker.internet.jwtAlgorithm() // 'RS512'
+   *
+   * @since 9.0.0
+   */
+  jwtAlgorithm(): string {
+    return this.faker.helpers.arrayElement(
+      this.faker.definitions.internet.jwt_algorithm
+    );
+  }
+
+  /**
+   * Generates a random JWT (JSON Web Token).
+   *
+   * @param options The optional options object.
+   * @param options.header The Header to use for the token. Defaults to a random object with the following fields: `alg` and `typ`.
+   * @param options.payload The Payload to use for the token. Defaults to a random object with the following fields: `iat`, `exp`, `nbf`, `iss`, `sub`, `aud`, and `jti`.
+   * @param options.refDate The date to use as reference point for the newly generated date.
+   *
+   * @see https://datatracker.ietf.org/doc/html/rfc7519
+   * @see faker.internet.jwtAlgorithm(): For generating random JWT (JSON Web Token) Algorithm.
+   *
+   * @example
+   * faker.internet.jwt()
+   * faker.internet.jwt({ header: { alg: 'HS256' }})
+   * faker.internet.jwt({ payload: { iss: 'Acme' }})
+   * faker.internet.jwt({ refDate: '2020-01-01T00:00:00.000Z' })
+   *
+   * @since 9.0.0
+   */
+  jwt(
+    options: {
+      /**
+       * The Header to use for the token. If present, it will replace any default values.
+       *
+       * @default
+       * {
+       *    alg = faker.internet.jwtAlgorithm();
+       *    typ = 'JWT'
+       * };
+       */
+      header?: Record<string, unknown>;
+      /**
+       * The Payload to use for the token. If present, it will replace any default values.
+       *
+       * @default
+       * {
+       *    iat = faker.date.recent();
+       *    exp = faker.date.soon();
+       *    nbf = faker.date.anytime();
+       *    iss = faker.company.name();
+       *    sub = faker.string.uuid();
+       *    aud = faker.string.uuid();
+       *    jti = faker.string.uuid();
+       * };
```

Comment:
All your default value are designed to be copy'pasted if read from the doc/JSDocs preview. The current syntax would be incorrect. At least to create an object that could be applied as an argument. Same goes for the header `@default` value.

```suggestion
       * {
       *    iat: faker.date.recent(),
       *    exp: faker.date.soon(),
       *    nbf: faker.date.anytime(),
       *    iss: faker.company.name(),
       *    sub: faker.string.uuid(),
       *    aud: faker.string.uuid(),
       *    jti: faker.string.uuid(),
       * };
```